AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -102,7 +102,7 @@ - (BOOL)deleteUserStorageWithAccountId:(NSString *)accountId {
 - (MSDocumentWrapper *)readWithPartition:(NSString *)partition
                               documentId:(NSString *)documentId
                             documentType:(Class)documentType
-                             readOptions:(MSReadOptions *)__unused readOptions {
+                             readOptions:(nullable MSReadOptions *)__unused readOptions {
   NSString *selectionQuery =
       [NSString stringWithFormat:@"SELECT * FROM \"%@\" WHERE \"%@\" = \"%@\" AND \"%@\" = \"%@\"", kMSAppDocumentTableName,
```

Comment:
the user table name is in this format, you should use a util method to get the table name
e.g. 
```
+ (NSString *)userTableNameWithAccountId:(NSString *)accountId {
  return [NSString stringWithFormat:kMSUserDocumentTableNameFormat, accountId];
}
```

Reply:
Will fix this in a different PR